src/pages/components/Overlay.qml

```diff
@@ -417,6 +417,11 @@ Background {
 
             onStatusChanged: browserPage.tabPageActive = (status == PageStatus.Active)
 
+            Component.onCompleted: {
```

Comment:
Place of ```Component.onCompleted``` (attached properties) has not been clearly defined, my personal preference for ```Component.onCompleted``` would be at the end of the ```tabPage```.

The ```positionViewAtIndex``` looks like obsolete now as this completes after tabView. Please remove if not really needed.